### DISCUSSION TITLE
Gitpod fail again ??

### DISCUSSION THREAD

**Discussion by rapi3** [benzino77/tasmocompiler#235] [General]
I tried yesterday and now to compile Tasmota but it fail to start compiling...
Is this the Gitpod fault ?

![Screenshot_2021-12-11_11-14-44](https://user-images.githubusercontent.com/18531150/145672935-cd20f297-b53c-4f37-be13-7b73033dda7e.png)

**Comment by rapi3** [comment]
to work it need to set again: PATH=$PATH:$HOME/.local/bin

**Comment by benzino77** [comment]
It should be fixed already. Have a look at this issue #232 and this commit: ad5ef693f8181c8e7823cbff8b74154fe52c3787.

Remove your workspace and start new one.

**Comment by benzino77** [comment]
I think there is still a problem when you try to use existing Workspace. Starting new one will work as expected.

**Comment by benzino77** [comment]
OK. I think it should work now when you start new workspace and also when open existing one.
